File: C/test/test_reverseDegree.c

#ifdef TEST

#include "unity.h"

#include "reverseDegree.h"

void setUp(void)
{
}

void tearDown(void)
{
}

void test_reverseDegree_Example1(void)
{
	char *input = "abc";
	int expected = 148;
	int actual = reverseDegree(input);
	TEST_ASSERT_EQUAL(expected, actual);
}

void test_reverseDegree_Example2(void)
{
	char *input = "zaza";
	int expected = 160;
	int actual = reverseDegree(input);
	TEST_ASSERT_EQUAL(expected, actual);
}
#endif // TEST

File: Rust/tests/test_reverse_degree.rs
use leetcode_rust::solution::Solution;

#[test]
fn test_example_1() {
    let input = "abc";
    let expected = 148;
    assert_eq!(Solution::reverse_degree(input.to_string()), expected);
}

#[test]
fn test_example_2() {
    let input = "zaza";
    let expected = 160;
    assert_eq!(Solution::reverse_degree(input.to_string()), expected);
}
